types.d.ts

```diff
@@ -0,0 +1,10 @@
+// This TypeScript modules definition file is generated by vaadin-maven-plugin.
+// You can not directly import your different static files into TypeScript,
+// This is needed for TypeScript compiler to declare and export as a TypeScript module.
+// It is recommended to commit this file to the VCS.
+// You might want to change the configurations to fit your preferences
+declare module '*.css' {
+    import { CSSResultGroup } from 'lit';
```

Comment:
Not sure is it ok for react?

Reply:
you can check this file https://github.com/vaadin/skeleton-starter-hilla-react/pull/18/files#diff-a150fb53afee945f23f3fe7c176007111f3ba81bb0d188af4077d4a0fc4caa37